platform/web/doc_classes/EditorExportPlatformWeb.xml

```diff
@@ -4,52 +4,85 @@
 		Exporter for the Web.
 	</brief_description>
 	<description>
+		The Web exporter customizes how a web build is handled. In the editor's "Export" window, it is created when adding a new "Web" preset.
+		[b]Note:[/b] Godot on Web is rendered inside a [code]&lt;canvas&gt;[/code] tag. Normally, the canvas cannot be positioned or resized manually, but otherwise acts as the main [Window] of the application.
 	</description>
 	<tutorials>
 		<link title="Exporting for the Web">$DOCS_URL/tutorials/export/exporting_for_web.html</link>
 		<link title="Web documentation index">$DOCS_URL/tutorials/platform/web/index.html</link>
 	</tutorials>
 	<members>
 		<member name="custom_template/debug" type="String" setter="" getter="">
-			Path to the custom export template. If left empty, default template is used.
+			File path to the custom export template used for debug builds. If left empty, the default template is used.
 		</member>
 		<member name="custom_template/release" type="String" setter="" getter="">
-			Path to the custom export template. If left empty, default template is used.
+			File path to the custom export template used for release builds. If left empty, the default template is used.
 		</member>
 		<member name="html/canvas_resize_policy" type="int" setter="" getter="">
-			The canvas resize policy determines how the canvas should be resized by Godot.
+			Determines how the canvas should be resized by Godot.
+			- [b]None:[/b] The canvas is not automatically resized.
+			- [b]Project:[/b] The size of the canvas is dependent on the [ProjectSettings].
+			- [b]Adaptive:[/b] The canvas is automatically resized to fit as much of the web page as possible.
 		</member>
 		<member name="html/custom_html_shell" type="String" setter="" getter="">
+			The custom HTML page that wraps the exported web build. If left empty, the default HTML shell is used.
```

Comment:
We could link to the docs on customizing the shelll: `$DOCS_URL/tutorials/platform/web/customizing_html5_shell.html`

Reply:
Would be much more beneficial if the tooltips could be clicked on, but it's better to mention it than not, I suppose.